### PR TITLE
fix "/" route

### PR DESCRIPTION
**Pull request by AP0rPi7uQ9**
This is a dumb method, but try not to modify the underlying code.

### CODE REVIEW

**Comment by achew22** [comment]
Thank you so much for sending this in, really excited to get this this fixed. Do you think you could add a test that demonstrates that you have solved the problem?

**Comment by AP0rPi7uQ9** [comment]
The test is in this link
https://github.com/grpc-ecosystem/grpc-gateway/issues/1909

File: runtime/mux.go [hunk 1]
```diff
@@ -180,6 +180,10 @@ func (s *ServeMux) Handle(meth string, pat Pattern, h HandlerFunc) {
 // HandlePath allows users to configure custom path handlers.
 // refer: https://grpc-ecosystem.github.io/grpc-gateway/docs/inject_router.html
 func (s *ServeMux) HandlePath(meth string, pathPattern string, h HandlerFunc) error {
+	if pathPattern == "/" {
+		index, _ := NewPattern(1, []int{2,0}, []string{""}, "")
+		s.Handle(meth, index, h)
```
Comment (johanbrandhorst) [inline runtime/mux.go:185]: I think this is missing a return below this line

**Review by johanbrandhorst — CHANGES_REQUESTED** [review]
Nice! Agree with Andrew, could you please add a test to the repository? Something like https://github.com/grpc-ecosystem/grpc-gateway/blob/master/runtime/mux_test.go#L425.

**Comment by AP0rPi7uQ9** [comment]
Okay, let me learn about adding tests and reply later.

File: runtime/mux.go (hunk 1 above)
Comment (AP0rPi7uQ9) [reply runtime/mux.go:185]: yes, forgot it.... return nil. i will fix it

**Comment by AP0rPi7uQ9** [comment]
Is this test correct?

And I don't know why this item keeps failing.

> golangci-lint / lint (pull_request) Failing after 54s — lint

**Comment by johanbrandhorst** [comment]
> Is this test correct?
> 
> And I don't know why this item always keeps failing.
> 
> > golangci-lint / lint (pull_request) Failing after 54s — lint

Yes, it's saying that the file changes you made aren't formatted correctly. You need to install and run `goimports` on the source code: https://pkg.go.dev/golang.org/x/tools/cmd/goimports

**Comment by johanbrandhorst** [comment]
Also, the CLA bot error is because you used a different account to make the test commit. You may want to make sure you're using the same github email in your commit as you have on your account.

**Comment by AP0rPi7uQ9** [comment]
> Also, the CLA bot error is because you used a different account to make the test commit. You may want to make sure you're using the same github email in your commit as you have on your account.

Thank you very much! you are very kind and glad to share with others.

**Comment by johanbrandhorst** [comment]
Hi, please rebase on `master`, we've changed our generation strategy quite a bit with https://github.com/grpc-ecosystem/grpc-gateway/pull/1971.

**Pull request closed without merging**